plugins/nf-wave/src/main/io/seqera/wave/plugin/WaveClient.groovy

```diff
@@ -397,6 +397,7 @@ class WaveClient {
         FROM ${config.condaOpts().mambaImage}
         COPY --chown=\$MAMBA_USER:\$MAMBA_USER conda.yml /tmp/conda.yml
         RUN micromamba install -y -n base -f /tmp/conda.yml && \\
+            micromamba install -y -n base conda-forge::procps-ng && \\
```

Comment:
I was thinking that instead of automagically adding dependencies in the container build it could be added an option to provide one or packages that should be a added in the resulting container e.g. 

```
wave.conda.basePackages = 'conda-forge::procps-ng'
```

or 

```
wave.conda.addPackages = 'conda-forge::procps-ng'
```

WDT?

Reply:
I like `wave.conda.basePackages`, I think it conveys the message.
Will implement as soon as I am finished with this week's duties.